modules/schematics/src/effect/files/__name@dasherize@if-flat__/__name@dasherize__.effects.ts.template

```diff
@@ -35,4 +35,22 @@ export class <%= classify(name) %>Effects {
 <% } else { %>
   constructor(private actions$: Actions) {}
 <% } %>
+
```

Comment:
If there was an option to extend a template that could be a nice solution here @brandonroberts @timdeschryver do you know if it's possible?

Reply:
Not currently